ocb3/src/lib.rs

```diff
@@ -79,17 +87,17 @@ type Sum = GenericArray<u8, SumSize>;
 impl<Cipher, NonceSize, TagSize> KeySizeUser for Ocb3<Cipher, NonceSize, TagSize>
 where
     Cipher: KeySizeUser,
-    TagSize: ArrayLength<u8> + IsLessOrEqual<U16>,
-    NonceSize: ArrayLength<u8> + IsLessOrEqual<U15>,
+    TagSize: ArrayLength<u8> + IsGreater<U0> + IsLessOrEqual<U16>,
+    NonceSize: ArrayLength<u8> + IsGreaterOrEqual<U6> + IsLessOrEqual<U15>,
```

Comment:
I think you also need to add bounds like `GrEq<NonceSize, U6>: NonZero,`. Plus it's worth to add compilation failure tests for unsupported sizes.

Reply:
It may be worth to add sealed `TagSizes`/`NonceSizes` traits to simplify public bounds.

Reply:
I can prepare a quick PR to amend this.

Reply:
We had those before although removed them in #590, though the requirements have gotten more complex since then. Still, it's a bounded range, and if we can express it properly with `typenum` it will be easier to eventually move to const generics.